[2024-01-23T16:59:18.328+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: tutorial_dag.not_valid scheduled__2024-01-23T15:30:00+00:00 [queued]>
[2024-01-23T16:59:18.355+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: tutorial_dag.not_valid scheduled__2024-01-23T15:30:00+00:00 [queued]>
[2024-01-23T16:59:18.356+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2024-01-23T16:59:18.373+0000] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): not_valid> on 2024-01-23 15:30:00+00:00
[2024-01-23T16:59:18.382+0000] {standard_task_runner.py:60} INFO - Started process 1315 to run task
[2024-01-23T16:59:18.393+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'tutorial_dag', 'not_valid', 'scheduled__2024-01-23T15:30:00+00:00', '--job-id', '9', '--raw', '--subdir', 'DAGS_FOLDER/tutorial_dag.py', '--cfg-path', '/tmp/tmpnqsjf6cu']
[2024-01-23T16:59:18.396+0000] {standard_task_runner.py:88} INFO - Job 9: Subtask not_valid
[2024-01-23T16:59:18.475+0000] {task_command.py:423} INFO - Running <TaskInstance: tutorial_dag.not_valid scheduled__2024-01-23T15:30:00+00:00 [running]> on host 825c5146cac7
[2024-01-23T16:59:18.556+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='tutorial_dag' AIRFLOW_CTX_TASK_ID='not_valid' AIRFLOW_CTX_EXECUTION_DATE='2024-01-23T15:30:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-23T15:30:00+00:00'
[2024-01-23T16:59:18.567+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-01-23T16:59:18.570+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', "echo 'Quantity NOT OK'"]
[2024-01-23T16:59:18.594+0000] {subprocess.py:86} INFO - Output:
[2024-01-23T16:59:18.625+0000] {subprocess.py:93} INFO - Quantity NOT OK
[2024-01-23T16:59:18.626+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-01-23T16:59:18.647+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=tutorial_dag, task_id=not_valid, execution_date=20240123T153000, start_date=20240123T165918, end_date=20240123T165918
[2024-01-23T16:59:18.687+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-01-23T16:59:18.699+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
